src/FileServiceManager/FileService/DAL/Interfaces/IStorage.cs

```diff
@@ -0,0 +1,21 @@
+﻿using FileService.Models;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace FileService.DAL
+{
+    public interface IStorage
+    {
+        Task<Guid> UploadFileAsync(IFormFile file, int companyId);
+
+        Task<IEnumerable<FileMetadata>> GetAllFilesAsync(int companyId);
```

Comment:
getting all files does not mean downloading them all, please consider excluding the content of file from the FileMetada class

Reply:
Fixed